generator/lib/ArtifactProviders/ExecutableSeed.ts

```diff
@@ -0,0 +1,48 @@
+import { ArtifactProvider } from './types'
+import * as swc from '@swc/core'
+import { TemplateInfo } from '~/src/templates'
+import { File } from '~/src/types'
+import { Index } from '~/src/utils'
+
+export const run: ArtifactProvider = <T extends Index<File>>(params: {
+  templateInfo: TemplateInfo
+  files: T
+}) => {
+  const prismaSeedScript = params.files['prisma/seed.ts']
+
+  if (!prismaSeedScript || prismaSeedScript.content === '') return []
+
+  const prismaSeedModule = transform({
+    content: prismaSeedScript.content,
+  })
+
+  return [
+    {
+      path: 'prisma/seed.js',
+      content: prismaSeedModule,
+    },
+  ]
+}
+
+const transform = ({ content }: { content: string }) => {
```

Comment:
Our transformer now is just regexp and then a pass of SWC to go from TS to JS. No MJS anymore either which simplifies things further.